paper-badge.html

```diff
@@ -19,6 +20,13 @@
 the anchor element specified in the `for` attribute, or, if that doesn't exist,
 centered to the parent node containing it.
 
+Badges can also contain an icon by adding the `icon` attribute and setting
+it to the id of the desired icon. Please note that you should still set the
+`label` attribute in order to keep the element accessible.
+
+`paper-badge` includes a default icon set. See [iron-iconset](../iron-iconset)
```

Comment:
So this doesn't work with the non-standard icons? What happens if the user of the badge (for example `demo/index.html`) imports the right icon file, and then passes something like `social:mood` as the icon attribute? Would that work?

I think it's going to be weird in the future if the badge only works with 10% of the icons -- i would think it should either work with some, or none...